app/forms/remote_search_form.rb

```diff
@@ -6,6 +6,8 @@ class RemoteSearchForm
     :checkbox,
     *SearchForm::TYPES_OF_ADVICE
 
+  validate :advice_methods_not_empty
```

Comment:
A small nit-pick but maybe `advice_methods_present` is clearer. Negating in naming is always an alarm bell for me.